### PR TITLE
Remove few tests from acceptance tag

### PR DESCRIPTION
**Pull request by Frawless**
Signed-off-by: Jakub Stejskal <xstejs24@gmail.com>

### Type of change

- Refactoring

Remove few tests from `acceptance` tag to make `acceptance` jobs shorter.

_Please describe your pull request_

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

**Review by scholzj — COMMENTED** [review]
Some other options I thought about:
* Should we drop `CruiseControlpiST.testCruiseControlBasicAPIRequests`. We still have `CruiseControlIsolatedST.testCruiseControlWithRebalanceResource` which is IMHO enough for acceptance.

File: systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/ListenersST.java [hunk 1]
```diff
@@ -443,7 +443,6 @@ void testOverrideNodePortConfiguration() {
     }
 
     @Test
-    @Tag(ACCEPTANCE)
```
[the remark this answers is not in the document]
Comment (Frawless) [reply systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/ListenersST.java:446]: NodePort test with acceptance tag is here https://github.com/strimzi/strimzi-kafka-operator/blob/master/systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/MultipleListenersST.java#L70

[the remark this answers is not in the document]
Comment (scholzj) [reply systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/ListenersST.java:446]: Ok, in that case we do not need it. I missed that one.

**Review by scholzj — APPROVED** [review]

**Pull request merged**